source/braille.py

```diff
@@ -27,73 +27,181 @@
 import brailleTables
 
 roleLabels = {
+	# Translators: Displayed in braille for an object which is a
+	# window.
+	controlTypes.ROLE_WINDOW: _("wnd"),
+	# Translators: Displayed in braille for an object which is a
+	# dialog.
+	controlTypes.ROLE_DIALOG: _("dlg"),
+	# Translators: Displayed in braille for an object which is a
+	# check box.
+	controlTypes.ROLE_CHECKBOX: _("chk"),
+	# Translators: Displayed in braille for an object which is a
+	# radio button.
+	controlTypes.ROLE_RADIOBUTTON: _("rbtn"),
 	# Translators: Displayed in braille for an object which is an
 	# editable text field.
 	controlTypes.ROLE_EDITABLETEXT: _("edt"),
 	# Translators: Displayed in braille for an object which is a
-	# list.
-	controlTypes.ROLE_LIST: _("lst"),
+	# button.
+	controlTypes.ROLE_BUTTON: _("btn"),
 	# Translators: Displayed in braille for an object which is a
 	# menu bar.
 	controlTypes.ROLE_MENUBAR: _("mnubar"),
 	# Translators: Displayed in braille for an object which is a
+	# menu item.
+	controlTypes.ROLE_MENUITEM: _("mnuitem"),
+	# Translators: Displayed in braille for an object which is a
 	# menu.
 	controlTypes.ROLE_POPUPMENU: _("mnu"),
 	# Translators: Displayed in braille for an object which is a
-	# button.
-	controlTypes.ROLE_BUTTON: _("btn"),
+	# combo box.
+	controlTypes.ROLE_COMBOBOX: _("cbo"),
 	# Translators: Displayed in braille for an object which is a
-	# check box.
-	controlTypes.ROLE_CHECKBOX: _("chk"),
+	# list.
+	controlTypes.ROLE_LIST: _("lst"),
 	# Translators: Displayed in braille for an object which is a
-	# radio button.
-	controlTypes.ROLE_RADIOBUTTON: _("rbtn"),
+	# graphic.
+	controlTypes.ROLE_GRAPHIC: _("gra"),
 	# Translators: Displayed in braille for an object which is a
-	# combo box.
-	controlTypes.ROLE_COMBOBOX: _("cbo"),
+	# help balloon.
+	controlTypes.ROLE_HELPBALLOON: _("hlp"),
+	# Translators: Displayed in braille for an object which is a
+	# tool tip.
+	controlTypes.ROLE_TOOLTIP: _("tltip"),
 	# Translators: Displayed in braille for an object which is a
 	# link.
 	controlTypes.ROLE_LINK: _("lnk"),
 	# Translators: Displayed in braille for an object which is a
-	# dialog.
-	controlTypes.ROLE_DIALOG: _("dlg"),
-	# Translators: Displayed in braille for an object which is a
 	# tree view.
 	controlTypes.ROLE_TREEVIEW: _("tv"),
 	# Translators: Displayed in braille for an object which is a
+	# tree view item.

```

Comment:
It's worth noting that this should rarely appear because we don't show the role for tree view items unless they provide no other info. Is it still worth including?